apps/system/js/multi_screen_controller.js

```diff
@@ -211,25 +158,34 @@
         }));
       });
     },
-    _displayListResultHandler: function(detail) {
-      if (!this.queryPromiseCallback) {
-        return;
-      }
 
-      switch (detail.type) {
-        case 'get-display-list-success':
-          this.queryPromiseCallback.resolve(detail.display);
-          break;
-        case 'get-display-list-error':
-          this.queryPromiseCallback.reject(detail.error);
+    _handle_mozPresentationChromeEvent: function(evt) {
+      this.debug('got mozPresentationChromeEvent event');
+
+      var detail = evt.detail;
+      switch(detail.type) {
+        case 'presentation-launch-receiver':
+          // target screen ID
+          this.screenId = this.requestDeviceId.split(':')[1];
```

Comment:
In the following code, the hash part should be an integer.
https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/remote/app.js#L30-L32

But `this.requestDeviceId` is like `hdmi:1`. So I need to extract the integer part. That's why I decide to use another variable to store it.